### PR TITLE
First draft of editorial policies.

### PR DESCRIPTION
**Pull request by mrshirts**
Created an editorial policy document, and have transferred some of the discussions in other parts of the site here.  This is a draft document.

### CODE REVIEW

**Comment by jchodera** [comment]
Do you really want a LaTeX/PDF for editorial policies if everything else is in web-ready readable Markdown? Why not leave it as Markdown?

**Comment by davidlmobley** [comment]
@jchodera - the thought was that more formal documents can be, well, more formally formatted and also posted as essentially editorial articles on the actual journal website (e.g. see the article currently at http://www.livecomsjournal.org/ if you scroll down).

**Comment by jchodera** [comment]
But you're not even using your own article template for that?

I'm a big fan of online editorial policies that are easy to read, find, search, and share without having to download a (usually out of date) PDF. If you're going to give up using your own template anyway, why not just write the thing in Markdown and convert it to PDF with any number of tools, like http://www.markdowntopdf.com/, [atom](https://atom.io/packages/markdown-pdf), or [pandoc](http://pandoc.org/demos.html). This would be infinitely easier to deal with.

Here's an example produced by http://www.markdowntopdf.com/: 
[README.pdf](https://github.com/livecomsjournal/journal_information/files/1275125/README.pdf)

**Comment by mrshirts** [comment]
> But you're not even using your own article template for that?

Well, we don't want it to be an 'article', since we don't want it with a doi, etc., doesn't have references, etc.  It's not that category.   But I could be convince other wise.

I'm also perfectly willing to go with Markdown if you're OK with it, David.  The markdown file would still be official, and have the various.  It would have to go on the website repository (to keep things in one place), but that's fine. 

We could actually remove the journal information repository and move it to the web site repository, but that doesn't need to be done now -- just an option.

**Comment by davidlmobley** [comment]
> But you're not even using your own article template for that?

My thought was that we would produce policies using our article template because it would look official. To me that's the main argument (aside from being able to post them on our official website) for having them in a PDF. 

@jchodera 
> I'm a big fan of online editorial policies that are easy to read, find, search, and share without having to download a (usually out of date) PDF.

Do you have an example of this? The journals I've looked at most often seem to usually have these as PDFs on letterhead, with a secondary option being as a normal web page on their official website. Having them in markdown on a secondary site seemed less "official" to me.

> I'm also perfectly willing to go with Markdown if you're OK with it, David. The markdown file would still be official, and have the various. It would have to go on the website repository (to keep things in one place), but that's fine.

That's fine with me I suppose.

**Comment by jchodera** [comment]
Sure! Compare these two author guides:

* PLoS: [guide for authors](http://journals.plos.org/plosone/s/submission-guidelines)
* JACS: [guide for authors](http://pubs.acs.org/page/jacsat/submission/authors.html)

I honestly can't even figure out which link(s) I am supposed to click on for the JACS guide. All of them? Just to find one detail?

**Comment by davidlmobley** [comment]
The JACS one is terrible. That's the worst I have ever seen. Ha. (It makes me glad I haven't even thought about submitting to JACS since I was in the Dill lab.) 

The PLOS one is great.

If we were able to lay out ours like theirs, I would be all for that. I love the navigation in the left hand side bar that unfolds as you scroll so you can easily see where you are. The way it's broken up in tables is also awesome. 

The other thing that I like about the PLOS one (which is harder for me to see how we will achieve it here) is that it's obviously an official journal document. I'm a little worried about how we're going to make that clear when we have our stuff in multiple places.

**Comment by mrshirts** [comment]
OK, I'll try a pass in the next few days to make it more like that.

**Comment by mrshirts** [comment]
Plus, of course, JACS doesn't take papers that have been preprints :)

**Comment by jchodera** [comment]
And that's why I am happy to help accelerate their demise.

File: editorial_material/editorial_policies.tex [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+\end{abstract}
+
+\section{Policies on Submissions to LiveCoMS}
+
+\subsection{Policy on Licensing}
+
+LiveCoMS does not ask for or request any copyright transfer.
+
+However, in order to submit to LiveCoMS, authors must provide, at
+minimum, a license for LiveCoMS to publish the article and distributed
```
Comment (davidlmobley) [inline editorial_material/editorial_policies.tex]: "distribute it" not "distributed it"

File: editorial_material/editorial_policies.tex [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+it free of charge. We recommend that the authors release the article
+under an open source license such as \href{Creative Commons
+  Attribution}{https://creativecommons.org/licenses/by/4.0/},
```
Comment (davidlmobley) [inline editorial_material/editorial_policies.tex]: "(also known as CC-BY)"??

File: editorial_material/editorial_policies.tex [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+releasing the document for anyone to copy and redistribute the
+material in any medium or format, and remix, transform, and build upon
+the material for any purpose, even commercially.  Making it available
+to all of course makes it possible for LiveCoMS to publish it, and for
+the community to edit and contribute.
```
Comment (davidlmobley) [inline editorial_material/editorial_policies.tex]: Add in closing, `"We highly recommend the CC-BY license in order to ensure your work can reach and help the broadest audience possible, and suggest that when considering the appropriate license you [read this analysis](http://openaccess.ox.ac.uk/2013/06/13/cc-by-what-does-it-mean-for-scholarly-articles-3/)"`.

File: editorial_material/editorial_policies.tex [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+Other more restrictive licenses may be possible as long as LiveCoMS
```
Comment (davidlmobley) [inline editorial_material/editorial_policies.tex]: "possible" -> "permissible"

File: editorial_material/editorial_policies.tex [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+has the permission to publish and excerpt from the document; this
+might be required if someone other than the authors has some rights to
+the material (for example, if it was previously published in some
+other journal). Generally, we only allow a more restrictive license in
+these cases, but are happy to discuss. Please ask the managing editors
+if you require some other licensing regime.
+
+\subsection{Policies on Author Order}
+Authors should, to the extent possible, determine the author order among themselves.
```
Comment (davidlmobley) [inline editorial_material/editorial_policies.tex]: Perhaps add, "In general, each work should have a section describing the actual contributions of authors (and of those acknowledged) to provide clarity. However, we acknowledge that institutions evaluating for merit, promotion, tenure or other cases may not read this level of detail, so traditional notions of author order (first author, corresponding author, etc.) may still be relevant and the authors will need to coordinate who should occupy these positions."

File: editorial_material/editorial_policies.tex [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+other journal). Generally, we only allow a more restrictive license in
+these cases, but are happy to discuss. Please ask the managing editors
+if you require some other licensing regime.
+
+\subsection{Policies on Author Order}
+Authors should, to the extent possible, determine the author order among themselves. 
+
+\subsection{Policy on Abandoned Documents}
+
+All submitters agree by submission of the document to so that if
```
Comment (davidlmobley) [inline editorial_material/editorial_policies.tex]: I would rephrase this paragraph entirely as it is too vague and does not actually specify how they have given consent. I think we want them to indicate they are explicitly agreeing. Perhaps something more like this:

"By the submission of their paper, document, or materials, all the authors consent that if they no longer are willing or able to maintain their work, LiveCoMS may assign another individual or individuals to do so, with appropriate modifications to the authors list. Authors would be given written notice (by e-mail and formal letter) if this were to happen and would have a period of six months to respond and/or designate a successor before LiveCoMS would do so for them. Typically, authors are *implicitly* giving a right to do this via licensing under \href{Creative Commons
 +  Attribution}{https://creativecommons.org/licenses/by/4.0/} or similar licenses which give others the right to create derivative works (potentially allowing others to ``resurrect'' a document which has been abandoned); however, we expect all authors to *explicitly* consent to this policy to avoid any confusion. Ordinarily, we expect this policy will be relevant only in unusual or extreme cases where an author or authors dies or leaves the field; in most other cases authors will presumably be available to designate their own successors or succession plan if a work is valuable to the field and will continue to need maintenance and the original author(s) are no longer willing or able to do so. However, we want to plan for the possibility of unusual events, hence our need for this policy."

File: editorial_material/editorial_policies.tex [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+case it is clear that anyone can reuse and ``resurrect'' the document.
+
+However, as a case of professional and academic courtesy, authors
+should At the same time, there are still issues of ``academic
+courtesy''. For this reason, if the document has not been updated by
+one of the authors, and they do not respond, submission to LiveCoMS
+means that the authors agree that the other others may continue. If no
+authors are contactable, then LiveCoMS may accept an article based on
+a previous LiveCoMS article as long as the previous authors are
+recognized.
```
Comment (davidlmobley) [inline editorial_material/editorial_policies.tex]: I would redo this paragraph too. What about: "Thus, for these reasons, authors submitting to LiveCoMS are agreeing that others may take over authorship of their article (with appropriate acknowledgment/recognition of the original authors) if they have abandoned their article as described above, and that LiveCoMS may accept revised versions of papers which have amended authorship in such circumstances."

File: editorial_material/editorial_policies.tex [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+means that the authors agree that the other others may continue. If no
+authors are contactable, then LiveCoMS may accept an article based on
+a previous LiveCoMS article as long as the previous authors are
+recognized. 
+
+\section{Policies About the Editorial Boards}
+
+\subsection{Policies on Decisions}
+Issues regarding governance structure, editorial board membership, leadership, and financial
+model must be decided by a 2/3 majority vote of the editorial
```
Comment (davidlmobley) [inline editorial_material/editorial_policies.tex]: "2/3 majority vote of the voting members of the editorial board"? Just to make clear we're not saying 2/3 of everyone but 2/3 of those voting....

**Review by davidlmobley — CHANGES_REQUESTED** [review]
@mrshirts - basically looks good to me with the suggested edits here; I think also we're going towards Markdown now?

**Comment by mrshirts** [comment]
Thanks for the large amount of feedback.  I will finish the PR with corrections/modifications later today, so that we have a record of that text here, and then convert to markdown and move to the website directory.

File: editorial_material/editorial_policies.tex (hunk 1 above)
Comment (mrshirts) [reply editorial_material/editorial_policies.tex]: done.

File: editorial_material/editorial_policies.tex (hunk 2 above)
Comment (mrshirts) [reply editorial_material/editorial_policies.tex]: Done.

File: editorial_material/editorial_policies.tex (hunk 4 above)
Comment (mrshirts) [reply editorial_material/editorial_policies.tex]: Done.

File: editorial_material/editorial_policies.tex (hunk 3 above)
Comment (mrshirts) [reply editorial_material/editorial_policies.tex]: Done.

File: editorial_material/editorial_policies.tex (hunk 5 above)
Comment (mrshirts) [reply editorial_material/editorial_policies.tex]: Done, though changed it from "should" to "must".  We can use the templates to provide guidance towards that.

File: editorial_material/editorial_policies.tex (hunk 6 above)
Comment (mrshirts) [reply editorial_material/editorial_policies.tex]: Done.

File: editorial_material/editorial_policies.tex (hunk 7 above)
Comment (mrshirts) [reply editorial_material/editorial_policies.tex]: Done.

File: editorial_material/editorial_policies.tex (hunk 8 above)
Comment (mrshirts) [reply editorial_material/editorial_policies.tex]: Done.

**Comment by mrshirts** [comment]
Merging as Mobley said he was OK after these drafts.  Will next convert to markdown, in two files, and move to the website.  

Will keep as separate markdown files for now, will consider merging back once we figure out whether we are merging the information from the other sections.
